homeassistant/components/prowl/notify.py

```diff
@@ -16,13 +16,18 @@
     ATTR_TITLE_DEFAULT,
     PLATFORM_SCHEMA as NOTIFY_PLATFORM_SCHEMA,
     BaseNotificationService,
+    NotifyEntity,
 )
+from homeassistant.config_entries import ConfigEntry
 from homeassistant.const import CONF_API_KEY
 from homeassistant.core import HomeAssistant
 from homeassistant.exceptions import HomeAssistantError
 from homeassistant.helpers import config_validation as cv
+from homeassistant.helpers.entity_platform import AddConfigEntryEntitiesCallback
 from homeassistant.helpers.typing import ConfigType, DiscoveryInfoType
 
+type ProwlConfigEntry = ConfigEntry[ProwlNotificationEntity]
+
```

Comment:
We're no longer storing anything in the `runtime_data`, so this is now misleading and not needed
```suggestion
```